www/docs/install.md

```diff
@@ -160,36 +159,29 @@ You can verify it using [our public key](https://goreleaser.com/static/gorelease
       ```
     1. Verify the signature:
       ```sh
-      cosign verify-blob \
-        -key https://goreleaser.com/static/goreleaser.pub \
-        -signature checksums.txt.sig \
+      COSIGN_EXPERIMENTAL=1 cosign verify-blob \
+        --signature checksums.txt.sig \
         checksums.txt
       ```
     1. If the signature is valid, you can then verify the SHA256 sums match with the downloaded binary:
       ```sh
       sha256sum --ignore-missing -c checksums.txt
       ```
 
-## Verifying docker images
+### docker images
 
-Our Docker image is signed with [cosign][].
+Our Docker images are signed with [cosign][].
```

Comment:
maybe we can give more information about the release, like how they can search GoReleaser binary SHA256 in the transparency log as `Tekton Chains` did its releases. WDYT?

* https://github.com/tektoncd/chains/releases/tag/v0.6.1

Reply:
Sgtm, can you add it to my branch or open a pr to it? Thankss